package.json

```diff
@@ -6,8 +6,11 @@
   "license": "BSD-3-Clause",
   "type": "module",
   "scripts": {
-    "dev": "pages dev",
-    "build": "vite build",
+    "dev": "yext pages build && yext pages generate-test-data && pages dev",
```

Comment:
For dev we don't need the artifact-output, so change `yext pages build` to `vite build` or `npm run build:local`. I'd prefer the latter for the same reason as in my below comment.